app/controllers/cards_controller.rb

```diff
@@ -40,7 +40,8 @@ def show
   end
 
   def preview
-
+    @external_contributions = @card.contributions.select { |contribution| contribution.user.nil? }
```

Comment:
@external_contributions has to be replaced by @contributions = @card.contributions